### PR TITLE
add templating, fill out README

### PR DESCRIPTION
**Pull request by mauleb**
## Changes

* added `<% projectName %>` to various places through the project
* define a better default `README` file

## Steps to Test
* [ ] read over the `README` changes
* [ ] verify `@jbknowledge/create-react-app` replaced all of the `<% projectName %>` tokens as intended

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,55 @@
...
+
+Each value in `env.js` can be overridden programmatically per environment via `npm run configure`. The `configure` script provided by `@jbknowledge/react-dev` will replace each key value in `window.env` with the value of `ENV_<key>` if it exists. For example, if you have the following `env.js` file:
+
+```js
+window.env = {
+  SOME_ARG: 'local value'
+};
+```
+
+And you run `npm run configure` while having the environment variable `ENV_SOME_ARG` set to `production value`. Your `env.js` file will be updated to:
```
Comment (joshytaco) [inline README.md:41]: @mauleb I think it would be a good idea to have an example of the command.

**Comment by joshytaco** [comment]
@mauleb Also add template string replacement for the `name` properties in `manifest.json`.

**Review by joshytaco — CHANGES_REQUESTED** [review]
Finished review.

File: README.md (hunk 1 above)
Comment (mauleb) [reply README.md:41]: that essentially is an example.

**Review by joshytaco — APPROVED** [review]

**Pull request merged**